.goreleaser.yaml

```diff
@@ -161,6 +161,33 @@ rigs:
     description: Deliver Go binaries as fast and easily as possible
     license: MIT
 
+aurs:
+  - homepage:  https://goreleaser.com
+    description: Deliver Go binaries as fast and easily as possible
+    maintainer: 'Carlos Alexandro Becker <carlos at becker dot software>'
+    contributors:
+      - 'Fernandez Ludovic <lfernandez dot dev at gmail dot com>'
+    license: MIT
+    private_key: '{{ .Env.AUR_KEY }}'
+    git_url: 'ssh://aur@aur.archlinux.org/goreleaser-bin.git'
+    package: |-
```

Comment:
Hi @caarlos0, Shoukd we really put installation code onto the goreleaser.yaml?

Reply:
we would need to put it somewhere... this way is analogous to `brews.install`...